### ISSUE TITLE
Y24-096 Tube Rack Support for additional Plate barcodes

### ISSUE THREAD

**Issue by yoldas** [sanger/limber#1703]
**User story**
As PSD, we would like to support for printing an additional plate barcode for the versioned tube racks proposed in Reuse of TubeRacks document.

**Who are the primary contacts for this story**
Abdullah, Katy, Andrew

**Who is the nominated tester for UAT**
TBD

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ]  Allow generating a plate barcode for a tube rack
- [ ] Allow finding the tube rack using both the tube rack barcode and the plate barcode
- [ ] Continue presenting as a tube rack in Limber

**Dependencies**
This story is blocked by the following dependencies:
- #1482

References
This story has a non-blocking relationship with:
- #1388

**Additional context**
The versioned tube rack model is researched in DPL-1011 Re-use of tube racks and the model is summarised in the Google Document https://docs.google.com/document/d/1rvPXeCk5rBEf72VkchHCD2FRmWANboke2pln_Lt_j5U/edit?usp=sharing

This story is part of re-implementation of DPL-914 Create tube rack containing banked PBMC tubes https://github.com/sanger/limber/issues/1388

**Comment by KatyTaylor** [comment]
I think this should work already?

**Comment by KatyTaylor** [comment]
How are you imagining this would look in the UI?

I think, when the epic Y24-113 is released, we could get the users to start using the etched barcodes rather than printing out plate barcode labels and sticking them on the rack - so we wouldn't need this story.

Another alternative is that for the first iteration, we don't save the etched barcode against the tube rack (when implementing Y24-104 Tube Rack Defrosting Uploads) - we keep using the plate barcode - and then there is no change for the user. We could then do a small story to save the tube rack barcode, which we could coordinate with the user.

Let me know what you think. Just trying to avoid having to make a UI change in the generic printing form, for a transient requirement.

**Comment by yoldas** [comment]
I have added the On Hold label to this story following Katy's comment.
